packages/resin-event-log/test/resin-event-log.spec.js

```diff
@@ -253,4 +279,128 @@ describe('ResinEventLog', function () {
 			})
 		})
 	})
+
+	describe('gosquared track', function () {
+		this.timeout(15000)
+		var endpoint = '/tracking/v1/event'
+		var eventLog
+
+		afterEach(function() {
+			return eventLog.end()
+		})
+
+		it('should make request to gosquared', function (done) {
+			var EXPECTED_EVENT = 'x'
+			var mockedRequest = createGsMock({
+				endpoint: endpoint,
+				method: 'POST',
+				filterBody: function (body, eventName) {
+					return (body.event.name === '[' + SYSTEM + '] ' + EXPECTED_EVENT)
+				}
+			})
+			eventLog = ResinEventLog({
+				gosquaredId: GOSQUARED_ID,
+				gosquaredApiKey: GOSQUARED_API_KEY,
+				prefix: SYSTEM,
+				debug: true,
+				afterCreate: function(err, type, jsonData, applicationId, deviceId) {
+					if (err) {
+						console.error('gosquared error:', err)
+					}
+					expect(!err).to.be.ok
+					expect(type).to.be.equal(EXPECTED_EVENT)
+
+					if (!IS_BROWSER) {
+						expect(mockedRequest.isDone()).to.be.ok
+						done()
+					} else {
+						// TODO: mock browser tests.
+						// see: https://github.com/resin-io-modules/resin-analytics/pull/14
+						done()
+					}
+				}
+			})
+
+			eventLog.start(FAKE_USER).then(function () {
+				eventLog.create(EXPECTED_EVENT)
+			})
+		})
+
+		it('should have semantic methods like device.rename that send requests to gosquared', function (done) {
+			var EXPECTED_EVENT = 'Device Rename'
+			var mockedRequest = createGsMock({
+				endpoint: endpoint,
+				filterBody: function (body, eventName) {
+					return (body.event.name === '[' + SYSTEM + '] ' + EXPECTED_EVENT)
+				}
+			})
+
+			eventLog = ResinEventLog({
+				gosquaredId: GOSQUARED_ID,
+				gosquaredApiKey: GOSQUARED_API_KEY,
+				prefix: SYSTEM,
+				debug: true,
+				afterCreate: function(err, type, jsonData, applicationId, deviceId) {
+					if (err) {
+						console.error('gosquared error:', err)
+					}
+					expect(!err).to.be.ok
+					expect(type).to.be.equal(EXPECTED_EVENT)
+
+					if (!IS_BROWSER) {
+						expect(mockedRequest.isDone()).to.be.ok
+						done()
+					} else {
+						// TODO: mock browser tests.
+						// see: https://github.com/resin-io-modules/resin-analytics/pull/14
+						done()
+					}
+				}
+			})
+
+			eventLog.start(FAKE_USER).then(function () {
+				eventLog.device.rename()
+			})
+		})
+
+		it('should attach person to event when user is given', function (done) {
+			var EXPECTED_EVENT = 'y'
+			var mockedRequest = createGsMock({
+				endpoint: endpoint,
+				filterBody: function (body, eventName) {
+					return (
+						body.event.name === '[' + SYSTEM + '] ' + EXPECTED_EVENT &&
+						body.person_id == FAKE_USER.id
```

Comment:
It can't because `user.id` is a number and the `body.person_id` is a string. I can do explicit type conversion but implicit is better in this case no?